src/cpp/shared/HandEigenData.h

```diff
@@ -1,6 +1,6 @@
 #pragma once 
 
-#include "../../shared/hand_eigen_model.h"
+#include "./hand_eigen_model.h"
```

Comment:
This is separate to the big rename?  Add to PR title?

Reply:
I've just found out that it began to break Linux build at some moment. That's why I fixed it.